internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -724,6 +727,19 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 									}
 								}
 							}
+							// check for log custom fields that need to be transformed
+							for _, logCustomFields := range logCustomFieldsTransform {
```

Comment:
Can we write a test for this?

Reply:
definitely a good idea. checkout https://github.com/cloudflare/cf-terraforming/blob/master/testdata/cloudflare/cloudflare_ruleset_zone_ddos_l7.yaml and https://github.com/cloudflare/cf-terraforming/tree/master/testdata/terraform/cloudflare_ruleset_zone_ddos_l7 for putting one together.

Reply:
I'll work on getting some tests added. Going to need to get the acceptance test zone entitled to use the new phase.

Reply:
these tests are all mocked so no real HTTP requests are made. if you know what the payload is, you can throw it into the file.